bundles/org.openhab.binding.nikobus/src/main/java/org/openhab/binding/nikobus/internal/handler/NikobusBaseThingHandler.java

```diff
@@ -20,15 +20,21 @@
 import org.eclipse.smarthome.core.thing.ThingStatusDetail;
 import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
 import org.openhab.binding.nikobus.internal.NikobusBindingConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * The {@link NikobusBaseThingHandler} class defines utility logic to be consumed by Nikobus thing(s).
  *
  * @author Boris Krivonog - Initial contribution
+ * @author Wouter Denayer - support for module addresses as seen in the Niko PC tool
  */
 @NonNullByDefault
 abstract class NikobusBaseThingHandler extends BaseThingHandler {
-    private @Nullable String address;
+    @Nullable
+    protected String address;
```

Comment:
Revert this change. @Nullable should be positioned as was.

Reply:
ok